index.html

```diff
@@ -15,9 +15,11 @@
 <body>
     <main>
         <div class="landingTopContainer">
-            <div class="landingLeft"></div>
+            <div class="stopwatchContainer">
+                <p class="stopwatchText">Stopwatch: <span id="stopwatchSecondsDisplay">0</span> seconds</p>
+            </div>
             <div class="landingTitleContainer">
-                <h1 class="landingTitle">SURFWAVE Fury</h1>
+                <h1 class="landingTitle">SURFWAVE FURY</h1>
```

Comment:
all caps should be achieved using css and the html should say "Surfwave Fury"

Reply:
Resolved - text transform on the class of the H1.